src/main.rs

```diff
@@ -108,6 +111,7 @@ pub async fn bootstrap(config: &CliArgs) -> DeployBridgeOutput {
     )
     .unwrap();
     log::info!("✅ Core setup init for L1 successful.");
+    sleep(Duration::from_secs(60)).await;
```

Comment:
what is this sleep for?

Reply:
actually madara needs a l1 core contract while starting, so I run bootstraper first, get the core contract, add it to madara code, run madara, meanwhile bootstraper is sleeping

I know this is not ideal and we have to think about how to improve the devX here